pyproject.toml

```diff
@@ -41,10 +41,12 @@ dependencies = [
     "json-repair>=0.30.0",
     "tenacity>=8.2.3",
     "anyio",
+    "mcp>=1.6.0",
```

Comment:
Please don't include mcp in the main dependency. Include it in the extra dependency instead